### ISSUE TITLE
cannot use tpm-fido when tpm has owner password set

### ISSUE THREAD

**Issue by monperrus** [psanford/tpm-fido#31]
Thanks for the great tool.

FYI, I notice that tpm-fido does not work when the owner password is set on the tpm.

**Comment by psanford** [comment]
That's correct. We could add support for accepting an owner password out of band from fido operations, but whatever that solution is will be fairly clunky.

Did you set the owner password on your tpm yourself, or was it set for you by an application or administrator?

**Comment by monperrus** [comment]
Thanks for the answer.

I'm admin on my machine, so I set myself both the owner and endorsement passwords.
